build.sh

```diff
@@ -191,7 +191,7 @@ case "$COMMAND" in
     # Clean
     ######################################
     "clean")
-        find . -type d -name build -ls -delete
+        find . -type d -name build -exec rm -r "{}" \;
```

Comment:
`rm -r "{}" +` is a little faster (passes as many filenames as possible to a single invocation of rm).